apps/reactotron-app/src/renderer/contexts/Standalone/useStandalone.ts

```diff
@@ -30,13 +35,16 @@ export interface Connection extends ReactotronConnection {
 }
 
 interface State {
+  serverStatus: ServerStatus
   connections: Connection[]
   selectedClientId: string
   orphanedCommands: any[] // Command[]
   commandListeners: ((command: any) => void)[] // ((command: Command) => void)[]
 }
 
 type Action =
+  | { type: ActionTypes.ServerStarted; payload: undefined }
+  | { type: ActionTypes.ServerStopped; payload: undefined }
```

Comment:
I honestly think we should implement MobX-State-Tree in Reactotron. I am not a big fan of the useReducer & provider/context DX.